args.go

```diff
@@ -287,11 +288,11 @@ func (a *Args) GetUfloatOrZero(key string) float64 {
 
 // GetBool returns boolean value for the given key.
 //
-// true is returned for '1', 'y' and 'yes' values,
+// true is returned for '1', 'y', 'yes', 'true', 't' and also their UPPERCASE analogs values,
 // otherwise false is returned.
 func (a *Args) GetBool(key string) bool {
-	switch string(a.Peek(key)) {
-	case "1", "y", "yes":
+	switch strings.ToLower(string(a.Peek(key))) {
```

Comment:
please get rid of ToLower and just add UPPERCASE analogs in the `case`. we should omit unnecessary allocations

Reply:
I didn't even notice the original pull request had this when I merged it into my fork :(
I removed the possible allocation and added a bunch of fixed cases including all the cases supported by `strings.ParseBool`.